_element-template/src/index.tsx

```diff
@@ -8,8 +8,6 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
 }
 
 const MyComponent: React.FC<Props> = ({ someText }) => {
-  // const { theme }: any = useThemeUI()
```

Comment:
this was left because you might want to use it in your real component

Reply:
It's an anti-pattern